pygsl/meson.build

```diff
@@ -0,0 +1,14 @@
+#: Todo: revisit if there is a more elegant way for this
+py_sources = run_command('find',
+['.',  '-maxdepth', '1', '-name', '*.py'],
+  check : true
+).stdout().strip().split('\n')
+
+py.install_sources(
+  py_sources,
+  pure: false,
+  subdir: 'pygsl'
+)
```

Comment:
- https://mesonbuild.com/Reference-manual_functions.html#install_subdir maybe?
- if you are going to use a Unix `find` here anyway then you might as well do it recursively, using `preserve_path: true` to prevent flattening subdirectories